packages/knip/src/plugins/node-modules-inspector/types.ts

```diff
@@ -0,0 +1,4 @@
+export type PluginConfig = {
+  plugins?: string[];
+  entryPathsOrPatterns?: string[];
+};
```

Comment:
I think this file should be either corrected (also please rename `PluginConfig` to something more descriptive) or removed.